.eslintrc.js

```diff
@@ -97,7 +97,14 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['**/__tests__/**', '**/*.test.js', '**/*.stories.js', '**/storyshots/**/stories/**'],
+      files: [
+        '**/__tests__/**',
+        '**/*.test.js',
+        '**/*.stories.js',
+        '**/storyshots/**/stories/**',
+        'docs/src/new-components/lib/StoryLinkWrapper.js',
```

Comment:
This line seems very specific? 😄

Reply:
Those are the ones that gave me grief, why generalize when you don't have to?